php-classes/Slate/CBL/DemonstrationSkill.php

```diff
@@ -22,6 +22,9 @@ class DemonstrationSkill extends \ActiveRecord
             'type' => 'tinyint',
             'unsigned' => true
         ]
+        ,'Demonstrated' => [
```

Comment:
@jmealo is this field supposed to be getting added to this model? Currently `Demonstrated` is a column in the `Demonstration` model table which is referenced by the DemonstrationID column in this table. I thought we solved this by adding Demonstration to the SQL added above in CompetenciesRequestHandler.php and scrapped the idea of duplicating the Demonstrated field from the Demonstration table to the DemonstrationSkill table?